### ISSUE TITLE
Is UrlLocalizationScheme.Scheme3 working and how?

### ISSUE THREAD

**Issue by asapostolov** [turquoiseowl/i18n#216]
I tried to use UrlLocalizationScheme.Scheme3 in my app but it throws exception. Is it implemented? if yes - what is the proper way to use it?

**Comment by turquoiseowl** [comment]
No, I never got round to implementing Scheme3. It's effectively a stub for when someone needs it and wants to code it.
